src/csharp/Microsoft.Spark.E2ETest/IpcTests/Sql/DataFrameReaderTests.cs

```diff
@@ -75,6 +76,11 @@ public void TestSignaturesV2_3_X()
             string textFile = $"{TestEnvironment.ResourceDirectory}people.txt";
             Assert.IsType<DataFrame>(dfr.Text(textFile));
             Assert.IsType<DataFrame>(dfr.Text(textFile, textFile));
+
+            string randomName = $"table_{new Random().Next(100000, 999999).ToString()}";
```

Comment:
Is this `ToString()` needed? I think this can be simplified.

Reply:
I've copied the test from Spark.Table